release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
```

Comment:
You might name this property just "releases-index" or even just "releases". I'd add another property for product called "product" with the value ".NET Core". This would enable us or others to use this same schema for other products.

Reply:
going with 'releases-index'